static/evchecker.html

```diff
@@ -0,0 +1,93 @@
+  </style>
+  <form action="javascript:void(0);" class="pure-form pure-form-stacked">
+    <fieldset>
+      <input id="target" placeholder="Target">
+      <br>
+      <input id="oid" placeholder="OID Policy">
+      <br>
+      <textarea id="rootCertificate" placeholder="Root Certificate (PEM format)" rows="20" cols="70"></textarea>
+      <br>
+      <button class="pure-button" onclick="send()">Submit</button>
+    </fieldset>
+  </form>
+  <p id="result"/>
+  <script>
+function checkResult(id) {
+    return fetch("/api/v1/results?id=" + id)
+	.then(function(response) {
+            return response.json().then(function(json) {
+		let completion_percent = json.completion_perc;
+		if (completion_percent !== 100) {
+		    console.log("Not done yet, running again");
```

Comment:
Also print a dot in the html to indicate to give visual feedback to the user that the scan is progressing. The command line tool does this.